### ISSUE TITLE
Can mentions cut through limits?

### ISSUE THREAD

**Issue by skyfaller** [mastodon/documentation#1212]
[The documentation for the Limit moderation action](https://docs.joinmastodon.org/admin/moderation/#limit-user) currently says:

> A limited account is hidden to all other users on that instance, except for its followers. All of the content is still there, and it can still be found via search, **mentions**, and following, but the content is invisible publicly.

What does "mentions" mean in this context? Does it mean that if someone @ mentions you from a limited server, you see their mention despite the limit? Or does it mean something else? Perhaps that if you go to mention an account from a limited server, their username appears in the dropdown of possible references?

**Comment by trwnh** [comment]
if someone mentions the account, you can discover its existence.

how should this be clarified, in your opinion?

**Comment by skyfaller** [comment]
Wait, so all this means is that if someone else mentions an account or post on a limited server, you can still click through to it?

I genuinely thought it meant @ mention spam would not be stopped by limiting. Here are a couple of threads illustrating my confusion:

- https://jawns.club/@skyfaller/110310539502716368
- https://jawns.club/@skyfaller/110311207115047495

I was confused because I was still receiving reports about spam from a server I had just limited, and I thought this line in the documentation might explain it. Maybe it took a while for the limit to take effect?

I guess I'd add a line like, "Limiting a server will prevent mentions from accounts on the limited server from notifying your users." Or perhaps, "Your users will no longer receive notifications when a limited server or account mentions them."

**Comment by trwnh** [comment]
oh, in that case, it's that i think mentions (or just direct statuses?) might always be delivered? i heard that once but haven't verified if that's (still) the case. lemme go look into this...

**Comment by trwnh** [comment]
nope, it seems like it was probably just reports coming in for old notifications. i think silencing a domain doesn't hide old notifications. it's possible there might be a bug somewhere, but that would have to be tested and reproduced first. mentions from silenced users should not generate notifications, nor will their statuses be visible in threads, regardless of whether it is a user silence or a domain silence.

**Comment by skyfaller** [comment]
Thank you for the clear answer! Do you think you could get the contents of your last post into the documentation somewhere? It would have been helpful to me to read this when I was trying to figure out the exact effect a limit / silence would have.
